setup.py

```diff
@@ -15,17 +15,17 @@
 
 print((find_packages(exclude=["contrib", "docs", "tests"])))
 setup(
-    name="duck-xchem",
+    name="openduck",
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
     version="0.1.0",
     description="Library for running Dynamic Undokcing",
     long_description=long_description,
     # The project's main homepage.
-    url="https://github.com/xchem/duck.git",
+    url="https://github.com/CBDD/openduck.git",
     # Author details
-    author="Anthony Bradley",
+    author="Anthony Bradley, Simon Bray, Mihaela Sminova, Alvaro Serrano",
```

Comment:
```suggestion
    author="Anthony Bradley, Simon Bray, Mihaela Sminova, Alvaro Serrano",
```
I would reverse the author order